geomesa-core/src/test/scala/org/locationtech/geomesa/core/iterators/MapAggregatingIteratorTest.scala

```diff
@@ -0,0 +1,346 @@
537584,-78.4913034335911 38.0619058528612,-78.4868925118207 38.066889139645,-78.486513103012 38.0671700907363,-78.4851136285658 38.0690434872368,-78.4805851960675 38.0666634146305,-78.4802984729844 38.0657417981749,-78.4788228617472 38.0655700882117,-78.4774278973046 38.0649697592028,-78.4761327734651 38.0667658280819,-78.4745427740881 38.0676149601349,-78.4748174301878 38.0686982688173,-78.4729253470527 38.0700538600024,-78.4725295078174 38.0702446954685,-78.4717311577653 38.0705250627835,-78.4709935743572 38.0697689534562,-78.4722083419684 38.0680235864848,-78.4714503964357 38.0679766441182,-78.4712491169607 38.0681480530514,-78.4695137245745 38.0683514820538,-78.4695249339711 38.0679499444058,-78.4682882003939 38.0681111118,-78.4681945631601 38.0676091543367,-78.4668418676903 38.0678226411344,-78.4662524286672 38.0668374753678,-78.4653139480169 38.0656097925943,-78.4644758509069 38.0648692093359,-78.465335243717 38.0630575427858,-78.4663085789463 38.0613277305874,-78.4664494470514 38.061327770253,-78.4673892136686 38.0595317061067,-78.4687248297256 38.0572798254838,-78.4698455345863 38.0556396474866,-78.4711807511563 38.0540501730135,-78.4724353916266 38.0526164977684,-78.4720860496955 38.0519420067951,-78.471652148566 38.0514699274272,-78.4715798628713 38.051341206666,-78.4714714256453 38.0511266382172,-78.4713991679905 38.0509264122871,-78.4713992611406 38.0506832841434,-78.4717790748334 38.0502706003002,-78.4720943686619 38.0499394554518,-78.4723560322113 38.0496394915541,-78.4724164669572 38.0494173839294,-78.4723360778499 38.0492303372208,-78.4722499477388 38.0489815322078,-78.4718600453255 38.0489146039865,-78.4713772585177 38.0488365585299,-78.4710017199293 38.0488052736595,-78.4705680597292 38.0486664959493,-78.4703872918519 38.0484662149579,-78.4701343302694 38.0478368848681,-78.4699356469289 38.0472075403136,-78.4698289155783 38.0470086499745,-78.4695339345729 38.0468371439102,-78.4693053388149 38.0469393179596,-78.468407168336 38.0466034706965,-78.4676917660222 38.0462473734836,-78.4661192632977 38.045714987292,-78.4653292114835 38.047182785812,-78.4651074150484 38.048250382979,-78.4647453057632 38.0482502765916,-78.4642891032747 38.048663190151,-78.4639468173163 38.0492436680877,-78.4638729309242 38.0495125312664,-78.4573545185689 38.0497324748685,-78.4570999398137 38.0492609003817,-78.4571048708452 38.0499860033047,-78.4572469354638 38.0504060779264,-78.458898424969 38.0510299931243,-78.4567841773194 38.0556302795318,-78.4561273995112 38.0553332011668,-78.4557521877327 38.0549991258425,-78.4555445617933 38.0547458706718,-78.4552967746776 38.0545217622445,-78.4550136860866 38.0541576231591,-78.4547304813727 38.053989517875,-78.4542703578379 38.053569291964,-78.454093163388 38.053765231087,-78.4543764171386 38.0538493662526,-78.4544118048045 38.0539053700334,-78.4546595354771 38.0541574891887,-78.4541635943456 38.0544373334645,-78.4548714741119 38.0550816694308,-78.4492025035279 38.0589437591411,-78.4480347940116 38.0574871002506))"
```

Comment:
I think so.  My push-back would be to factor this out and only have it one place in the test code base.  

*shrugs*  No big deal; either way is fine.